### ISSUE TITLE
List control and Virtualized Table should be accessible

### ISSUE THREAD

**Issue by nmgokhale** [elyra-ai/canvas#437]
Virtualized table is a reusable component in List Control.

- Fix all the a11y violations in virtualized table.
- Move row buttons should be Icon only buttons. This will change their color from blue to black as shown in image below
-  Tooltip should be set to left side of the move row button.
<img width="254" alt="move row buttons tooltip " src="https://user-images.githubusercontent.com/25124000/104046981-bf83e080-5195-11eb-8ae2-0aba8f972b86.png">
